src/Main.hs

```diff
@@ -87,8 +87,15 @@ isInteractive = do
   return $ not $ null i
 
 main = do
-  -- Issue #201: disable backend when run in interactive mode
-  isInt <- isInteractive
-  if isInt
-    then runAgda [Backend backend{isEnabled = const False}]
-    else runAgda [Backend backend]
+  args <- getArgs
+
+  -- Issue #370: `agda2hs locate` will print out the path to the prelude agda-lib file
+  if args == ["locate"] then 
+    getDataFileName "agda2hs.agda-lib" >>= putStrLn
```

Comment:
I suggest using `putStrLn =<< getDataFileName "agda2hs.agda-lib"` for readability.